olive/passes/pytorch/common.py

```diff
@@ -43,10 +43,30 @@ def inherit_pytorch_from_hf(
 ) -> PyTorchModelHandler:
     # keep original io_config if present otherwise use the hf onnx_config
     io_config = model.to_json()["config"].get("io_config")
+    if not io_config and model.io_config:
+        # must have come from the automatic io config using optimum
+        hf_io_config = deepcopy(model.io_config)
+        hf_dummy_inputs = model.get_dummy_inputs()
+
+        # kv cache will be handled by the kv_cache flag in io_config
+        io_config = {
+            "input_names": [i for i in hf_io_config.get("input_names", []) if not i.startswith("past_key_values")],
+            "input_shapes": [],
+            "input_types": [],
+            "output_names": [o for o in hf_io_config.get("output_names", []) if not o.startswith("present")],
+            "dynamic_axes": {
+                k: v
+                for k, v in hf_io_config.get("dynamic_axes", {}).items()
+                if not k.startswith("present", "past_key_values")
```

Comment:
Maybe you missed to convert the argument to tuple?

Reply:
Done! I followed the lint error but did not see that the argument needed to be a tuple. 

Also improved the test asserts